frontend/components/TableContainer/DataTable/HeaderCell/HeaderCell.tsx

```diff
@@ -20,8 +24,21 @@ const HeaderCell = ({
     sortArrowClass = "ascending";
   }
 
+  let lastColumnHeaderWithTooltipClass = "";
+  // Since value will only be a string or a TooltipWrapper, this checks for a TooltipWrapper
+  // TODO - add better checking for this
```

Comment:
@fleetdm/frontend, can folks think of a more explicit way of checking that `value` is an instance of TooltipWrapper? This works since only a `string` or `TooltipWrapper` are passed in, but if that were to change in the future this would cause a bug. I've tried a few things but would appreciate folks' input!

Reply:
@jacobshandling could we just apply the class last-column-header for all last column headers? Whether it's string or jsx.element?

Reply:
I don't think the string will be affected much if we add that class to it?